bin/qbatch

```diff
@@ -30,7 +30,10 @@ OPTIONS = [os.environ.get("QBATCH_OPTIONS")] if os.environ.get(
     "QBATCH_OPTIONS") else []
 
 if os.getenv('SGE_ROOT'):
-    SYSTEM = os.environ.get("QBATCH_SYSTEM", "sge")
+    if os.environ.get("QBATCH_SYSTEM", "sge") or os.environ.get("QBATCH_SYSTEM", "slurm"):
+        SYSTEM = os.environ.get("QBATCH_SYSTEM")
+    else:
+        sys.exit("specify either sge or slurm for QBATCH_SYSTEM")
```

Comment:
I don't understand why you have slurm and SGE together here -- I've never used slurm, so forgive my ignorance. I would have expected something like: 

```python
if os.getenv('SGE_ROOT'):
    SYSTEM = os.environ.get("QBATCH_SYSTEM", "sge")
elif os.getenv('SOME_SLURM_ENVVAR'):
    SYSTEM = os.environ.get("QBATCH_SYSTEM", "slurm")
...
```

In any case, thank you for working on this!